awx/api/filters.py

```diff
@@ -131,6 +131,8 @@ def get_field_from_lookup(self, model, lookup):
                     new_parts.append(name_alt)
                 else:
                     field = model._meta.get_field(name)
+                if 'auth' in name or 'token' in name:
```

Comment:
@AlanCoding @rooftopcellist @chrismeyersfsu can we think of any false positives this might hit?  I figured it made more sense to just have a blanket ban on fields w/ these terms to protect ourselves in the future (and to avoid missing one).

Reply:
The only false positive I would be concerned about is if we allowed indexed searching on JSON fields, so that users would be creating host filters like `variables__platform__token=abced`, where platform and token are keys inside their variables dictionary.

But I don't think that's a thing yet.

Reply:
I think you could also make an argument for filtering those similarly - if you've got a host filter with a key that contains "auth" or "token", it's _probably_ a sensitive value, too.

Reply:
It looks like this will also prevent searching on SSO models, which imo is a plus.

Reply:
Agreed, cross-app filters should be disallowed. Think of settings...